### PR TITLE
test(plugin-meetings): added test for hydra roomId on new meetingInfo Api

### PR DESCRIPTION
**Pull request by emanuallan**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-260888

at the time of this writing: the test works when using v2, but other existing tests (16) fail when using v2

### CODE REVIEW

**Comment by arun3528** [comment]
This PR should be good to merge as its testing the old flow making sure the meetingInfo fetch works with room ID 

When the unified space meeting is enabled it will automatically check if the roomID logic is working , we dont have to add two test to check one with and one with out toggle 

We can test it when we are enabling this for everyone

File: packages/node_modules/@webex/plugin-meetings/test/utils/webex-config.js [hunk 1]
```diff
@@ -56,7 +56,10 @@ Config.webex = {
     reconnection: {
       enabled: true
     },
-    enableRtx: true
+    enableRtx: true,
+    experimental: {
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/test/utils/webex-config.js:60]: remove this and we will handle the unified space meeting toggle enablement by default in a seperate PR

**Review by arun3528 — APPROVED** [review]

**Comment by taymoork2** [comment]
Test itself is passing, errors are flakiness from another suite
Screen Shot 2021-11-29 at 9 26 22 AM

**Review by jonniespratley — APPROVED** [review]
Changed look good, very minimal which is best.

**Pull request merged**